config/config.go

```diff
@@ -11,13 +14,28 @@ var (
 	SDStoreURL string
 	// SDToken is SD_TOKEN value
 	SDToken string
+	// SDArtifactsDir is SD_ARTIFACTS_DIR value
+	SDArtifactsDir string
+	// BaseCommandPath is path of installing binary command
+	BaseCommandPath = "/opt/sd/commands/"
 )
 
+func addSlash(val string) string {
+	if val == "" {
+		return val
+	}
+	if val[len(val)-1] != '/' {
```

Comment:
It can be write with one `if` statement.
`if val != "" && val[len(val)-1] != '/' {`

Reply:
Surely. I will fix.